charter.html

```diff
@@ -285,13 +296,18 @@ <h2 id=work-items>Work Items</h2>
 <p>The Chairs may add Work Items, but must not add Work Items which lack
 the support of at least two <a href=#implementer>implementers</a>.
 
-<p class=note>Typically, Work Items begin life as a Proposal before
-being adopted.
+<p>Each Work Item should be accompanied by an explainer describing its
+proposed changes to the web platform. Editors should keep the Work
+Item's explainer up-to-date with the Work Item as it evolves.
+
+<p class=note>Since Work Items typically begin life as a Proposal before
+being formally adopted, they usually start out with an explainer.
 
 <p>When a Work Item's Editors deem the Work Item ready for migration to
 a standards body, they will notify the Chairs. The Editors and Chairs
-will then work together to identify the best destination, and will work
-with the destination standards body to successfully migrate the work.
+will then produce a Final Community Group Report and identify the best
```

Comment:
Do we really want to produce a Final CG report in all cases? Could be signing up for a lot of overhead for some things that go into various places/groups as a final home. Perhaps a "may" requirement instead?

Reply:
Fixed in 6bb7532.